node/core/tests/test_node_client.py

```diff
@@ -38,16 +37,17 @@ def test_send_signed_change_request(
 def test_send_scr_to_address_integration(
     test_server_address, regular_node_declaration_signed_change_request, smart_mocked_node_client
 ):
-    assert BlockchainFacade.get_instance().get_next_block_number() == 1
-    assert not ORMNode.objects.filter(_id=regular_node_declaration_signed_change_request.signer).exists()
+    blockchain_facade = BlockchainFacade.get_instance()
+    assert blockchain_facade.get_next_block_number() == 1
+    assert blockchain_facade.get_node_by_identifier(regular_node_declaration_signed_change_request.signer) is None
 
     client = smart_mocked_node_client
     scr = regular_node_declaration_signed_change_request
     response = client.send_signed_change_request(test_server_address, scr)
     assert response.status_code == 201
 
     assert BlockchainFacade.get_instance().get_next_block_number() == 2
-    assert ORMNode.objects.filter(_id=regular_node_declaration_signed_change_request.signer).exists()
+    assert blockchain_facade.get_node_by_identifier(regular_node_declaration_signed_change_request.signer) is not None
```

Comment:
@olegtropinin why changed? why can't we keep both asserts? the same questions for all other cases?

Reply:
I have kept both asserts.